README.md

```diff
@@ -12,55 +12,44 @@ inference.
 
     ```
-    git clone https://github.com/opencv/openvino_training_extensions.git
-    cd openvino_training_extensions
+    git clone https://github.com/openvinotoolkit/training_extensions.git
+    cd training_extensions
     ```
 
 2. Install prerequisites by running the following:
 
     ```
     sudo apt-get install libturbojpeg python3-tk python3-pip virtualenv
     ```
-
+3. Download and install [OpenVINO™](https://software.intel.com/en-us/openvino-toolkit) to export and efficiently run trained models.
+4. Install `ote` package:
+    ```
+    pip3 install -e .
+    ```
 
 # Models
 
-* [PyTorch\*](pytorch_toolkit)
-
-  * [Action Recognition](pytorch_toolkit/action_recognition)
-  * [Action Recognition 2](pytorch_toolkit/action_recognition_2)
-    - [Gesture Recognition](pytorch_toolkit/action_recognition_2/model_templates/gesture-recognition)
-  * [Object Re-Identification](pytorch_toolkit/object_reidentification)
-    - [Face Recognition](pytorch_toolkit/object_reidentification/face_recognition)
-    - [Person Re-Identification](pytorch_toolkit/object_reidentification/person_reidentification)
-    - [Vehicle Re-Identification](pytorch_toolkit/object_reidentification/vehicle_reidentification)
-  * [Human Pose Estimation](pytorch_toolkit/human_pose_estimation)
-  * [Instance Segmentation](pytorch_toolkit/instance_segmentation)
-  * [Machine Translation](pytorch_toolkit/machine_translation)
-  * [Object Detection](pytorch_toolkit/object_detection)
-    - [Face Detection](pytorch_toolkit/object_detection/model_templates/face-detection)
-    - [Horizontal Text Detection](pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/)
-    - [Person Detection](pytorch_toolkit/object_detection/model_templates/person-detection/)
-    - [Person Vehicle Bike Detection](pytorch_toolkit/object_detection/model_templates/person-vehicle-bike-detection)
-    - [Vehicle Detection](pytorch_toolkit/object_detection/model_templates/vehicle-detection)
-  * [Eye State Classification](pytorch_toolkit/open_closed_eye)
-  * [Segmentation of Thoracic Organs](pytorch_toolkit/segthor)
-  * [Super Resolution](pytorch_toolkit/super_resolution)
-  * [Text Spotting](pytorch_toolkit/text_spotting)
-  * [Question Answering](pytorch_toolkit/question_answering)
-
-* [TensorFlow\*](tensorflow_toolkit)
-
-  * [Action Detection](tensorflow_toolkit/action_detection)
-  * [Image Retrieval](tensorflow_toolkit/image_retrieval)
-  * [License Plate Recognition](tensorflow_toolkit/lpr)
-  * [Text Detection](tensorflow_toolkit/text_detection)
-  * [Text Recognition](tensorflow_toolkit/text_recognition)
-  * [Vehicle Attributes](tensorflow_toolkit/vehicle_attributes)
-  * [Bert](tensorflow_toolkit/bert)
-
----
-\* Other names and brands may be claimed as the property of others.
+After installation, you are ready to train your own models, evaluate and use
+them for prediction.
+
+* [Action Recognition](models/action_recognition_2)
```

Comment:
Maybe it's good time to rename it back to 'models/action_recognition' to resolve possible questions like `where is the first action_recognition here`?

Reply:
definitely, but not right now, I would minimize changes in single PR as much as possible